core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -79,7 +79,6 @@ ${h.initPageVariables(context)}
 
```

Comment:
You left `norefresh` undeprecated in this file! The one which is actually used, unlike `html.jelly`.